spec.md

```diff
@@ -107,7 +107,7 @@ platform/vendor specific protocols (AWS Kinesis, Azure Event Grid).
 
 The following abstract data types are available for use in attributes.
 
-- `Integer` - A 32-bit whole number.
+- `Integer` - A whole number in the range −2,147,483,647 to +2,147,483,647 inclusive. This is the range of a signed 32-bit twos-complement representation. Event formats do not have to use this encoding, but they MUST only use `Integer` values in this range.
```

Comment:
Can you wrap this at 80 columns - so it matches the rest of the doc?

Reply:
https://en.wikipedia.org/wiki/32-bit shows it as "...,648" not "...,647" for the low range.

Reply:
Thanks, fixed both. All green now :)

Reply:
thanks